packages/x-date-pickers/src/StaticDateTimePicker/StaticDateTimePicker.tsx

```diff
@@ -38,7 +38,7 @@ type StaticDateTimePickerComponent = (<TDate>(
  *
  * Demos:
  *
- * - [Date Time Picker](https://mui.com/components/x/react-date-pickers/date-time-picker/)
+ * - [Date Time Picker](https://mui.com/x/react-date-pickers/date-time-picker/)
```

Comment:
In MUI Core, I believe these lines of code are generated automatically and kept in sync in each PR.

Reply:
We will probably have improvements to do for this part indeed